pkg/features/features.go

```diff
@@ -20,9 +21,13 @@ var (
 		DBFSMetricByPlugin: {Default: false, PreRelease: featuregate.Alpha},
 		DBFSADController:   {Default: false, PreRelease: featuregate.Alpha},
 	}
+	defaultUpdatedOssfsVersionFeatureGate = map[featuregate.Feature]featuregate.FeatureSpec{
```

Comment:
```suggestion
	defaultOSSFeatureGate = map[featuregate.Feature]featuregate.FeatureSpec{
```
I think all OSS feature gates can goes here, not just this one.